R/geom-text.R

```diff
@@ -31,8 +31,8 @@
 #'
 #' @section Alignment:
 #' You can modify text alignment with the `vjust` and `hjust`
-#' aesthetics. These can either be a number between 0 (right/bottom) and
-#' 1 (top/left) or a character (`"left"`, `"middle"`, `"right"`, `"bottom"`,
```

Comment:
There's no problem with this fix, but I personally prefer "left/top" to match the order of "right/bottom". What do you think?

Reply:
I like the order matching part, but it should be left/bottom & right/top then

Reply:
Ah, sorry, you are right. For some reason, I looked at the old description...